example/src/views/login.js

```diff
@@ -16,20 +16,7 @@ export default class Login extends Component {
           ref={(snapScroll) => { this.snapScroll = snapScroll; }}
           horizontal={true}
           decelerationRate={0}
-          onResponderRelease={()=>{
-            var interval = SCREEN_WIDTH; // WIDTH OF 1 CHILD COMPONENT
-
-            var snapTo = (this.scrollingRight)? Math.ceil(this.lastx / interval) :
-              Math.floor(this.lastx / interval);
-            var scrollTo = snapTo * interval;
-            this.snapScroll.scrollTo({x: scrollTo, y: 0, animated: true});
-          }}
-          scrollEventThrottle={32}
-          onScroll={(event)=>{
-            var nextx = event.nativeEvent.contentOffset.x;
-            this.scrollingRight = (nextx > this.lastx);
-            this.lastx = nextx;
-          }}
+          pagingEnabled
```

Comment:
@Monte9 That was not working on Android. Was there any reason why the pagination was sooo complicated, and have I broken something ? 🤔